libs/feature/record/src/lib/data-view/data-view.component.html

```diff
@@ -8,15 +8,13 @@
     [choices]="choices"
     (selectValue)="selectLink($event)"
   ></gn-ui-dropdown-selector>
-  <ng-container *ngIf="hidePreview; else dataView">
+  <ng-container *ngIf="hidePreview && mode === 'chart'; else dataView">
```

Comment:
Integrating this logic into the `selectLink` method could keep the template simpler. WDYT?